src/style.css

```diff
@@ -0,0 +1,173 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: local("Avenir-Book"),
+    url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: local("Avenir-Heavy"),
+    url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  /* font-weight: 300; */
+}
+
+.bigField {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.firstBar {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  vertical-align: middle;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
```

Comment:
You don't need to add it here. You already have it in the body tag. Check it everywhere